catalogs/catalog-hadoop/build.gradle.kts

```diff
@@ -101,6 +103,9 @@ tasks {
       exclude("javax.servlet-*.jar")
       exclude("kerb-*.jar")
       exclude("kerby-*.jar")
+      // As L55-56 takes no effect actually in the final distribution package, we need to exclude
```

Comment:
Please don't use `L55-56` the style, because the line number  will change.

Reply:
Maybe some other jars introduce `protobuf-java` dependences too

Reply:
Resolved